w_common/lib/src/browser/disposable_browser.dart

```diff
@@ -232,27 +232,27 @@ class Disposable implements disposable_common.Disposable {
 
   @override
   StreamSubscription<T> listenToStream<T>(
-          Stream<T> stream, void onData(T event),
-          {Function onError, void onDone(), bool cancelOnError}) =>
+          Stream<T> stream, void onData(T event)?,
+          {Function? onError, void onDone()?, bool? cancelOnError}) =>
       _disposable.listenToStream(stream, onData,
           onError: onError, onDone: onDone, cancelOnError: cancelOnError);
 
   @override
-  T manageAndReturnTypedDisposable<T extends disposable_common.Disposable>(
-          T disposable) =>
+  T? manageAndReturnTypedDisposable<T extends disposable_common.Disposable>(
+          T? disposable) =>
```

Comment:
This should not accept a nullable disposable and should return a non-nullable disposable. Same thing on line 250 and 255

Reply:
Unfortunatly, `manageDisposable` does accept `null` values [here](https://github.com/Workiva/w_common/blob/934e88599339cc209183366b47a9709298239df2/w_common/lib/src/common/disposable.dart#L556)

Reply:
I don't think it should. I think you should make `manageDisposable` non-nullable and get rid of the `null` assertion.